   186.7ms (setup:858): 2025-01-18 06:57:58
   186.8ms (setup:859): /usr/bin/pipx install flask
   186.9ms (setup:860): pipx version is 1.4.3
   186.9ms (setup:861): Default python interpreter is '/usr/bin/python3'
   186.9ms (mkdir:74): creating directory /home/ubuntu/.local/share/pipx/venvs
   187.1ms (mkdir:74): creating directory /home/ubuntu/.local/bin
   187.1ms (mkdir:74): creating directory /home/ubuntu/.local/share/man
   188.1ms (mkdir:74): creating directory /home/ubuntu/.cache/pipx
   188.2ms (setup:870): Adding CACHEDIR.TAG to cache directory
   188.9ms (package_name_from_spec:370): Determined package name: flask
   189.0ms (package_name_from_spec:371): Package name determined in 0.0s
   189.4ms (run_subprocess:168): running /usr/bin/python3 -m venv --without-pip /home/ubuntu/.local/share/pipx/venvs/flask
   253.0ms (run_subprocess:190): stdout:
   253.0ms (run_subprocess:192): stderr:
   253.0ms (run_subprocess:193): returncode: 0
   253.7ms (run_subprocess:168): running /usr/bin/python3 -m venv --clear /home/ubuntu/.local/share/pipx/shared
  3746.0ms (run_subprocess:190): stdout:
  3746.1ms (run_subprocess:192): stderr:
  3746.1ms (run_subprocess:193): returncode: 0
  3746.5ms (run_subprocess:168): running <checking pip's availability>
  3766.9ms (run_subprocess:190): stdout: ModuleSpec(name='pip', loader=<_frozen_importlib_external.SourceFileLoader object at 0x7ab2e13ffc50>, origin='/home/ubuntu/.local/share/pipx/shared/lib/python3.12/site-packages/pip/__init__.py', submodule_search_locations=['/home/ubuntu/.local/share/pipx/shared/lib/python3.12/site-packages/pip'])
  3767.0ms (run_subprocess:193): returncode: 0
  3767.1ms (upgrade:98): Upgrading shared libraries in /home/ubuntu/.local/share/pipx/shared
  3767.4ms (run_subprocess:168): running /home/ubuntu/.local/share/pipx/shared/bin/python -m pip --no-input --disable-pip-version-check install --force-reinstall -q --upgrade pip >= 23.1
  5649.6ms (run_subprocess:190): stdout:
  5649.7ms (run_subprocess:192): stderr:
  5649.7ms (run_subprocess:193): returncode: 0
  5650.1ms (run_subprocess:168): running /home/ubuntu/.local/share/pipx/venvs/flask/bin/python -c import sysconfig; print(sysconfig.get_path('purelib'))
  5670.3ms (run_subprocess:190): stdout: /home/ubuntu/.local/share/pipx/venvs/flask/lib/python3.12/site-packages
  5670.4ms (run_subprocess:193): returncode: 0
  5670.5ms (run_subprocess:168): running /home/ubuntu/.local/share/pipx/shared/bin/python -c import sysconfig; print(sysconfig.get_path('purelib'))
  5690.0ms (run_subprocess:190): stdout: /home/ubuntu/.local/share/pipx/shared/lib/python3.12/site-packages
  5690.1ms (run_subprocess:193): returncode: 0
  5690.3ms (run_subprocess:168): running /home/ubuntu/.local/share/pipx/venvs/flask/bin/python --version
  5691.9ms (run_subprocess:190): stdout: Python 3.12.3
  5692.0ms (run_subprocess:192): stderr:
  5692.0ms (run_subprocess:193): returncode: 0
  5692.2ms (_parsed_package_to_package_or_url:137): cleaned package spec: flask
  5693.5ms (run_subprocess:168): running /home/ubuntu/.local/share/pipx/venvs/flask/bin/python -m pip --no-input install flask
  7139.5ms (run_subprocess:193): returncode: 0
  7140.1ms (run_subprocess:168): running <fetch_info_in_venv commands>
  7165.4ms (run_subprocess:190): stdout: {"sys_path": ["/usr/lib/python312.zip", "/usr/lib/python3.12", "/usr/lib/python3.12/lib-dynload", "/home/ubuntu/.local/share/pipx/venvs/flask/lib/python3.12/site-packages", "/home/ubuntu/.local/share/pipx/shared/lib/python3.12/site-packages"], "python_version": "3.12.3", "environment": {"implementation_name": "cpython", "implementation_version": "3.12.3", "os_name": "posix", "platform_machine": "x86_64", "platform_release": "6.8.0-1021-aws", "platform_system": "Linux", "platform_version": "#23-Ubuntu SMP Mon Dec  9 23:59:34 UTC 2024", "python_full_version": "3.12.3", "platform_python_implementation": "CPython", "python_version": "3.12", "sys_platform": "linux"}}
  7165.5ms (run_subprocess:193): returncode: 0
  7184.2ms (get_venv_metadata_for_package:339): get_venv_metadata_for_package: 44ms
  7184.8ms (_parsed_package_to_package_or_url:137): cleaned package spec: flask
  7185.4ms (_symlink_package_resource:168): ⚠️  Note: flask was already on your PATH at /usr/bin/flask
  7186.1ms (run_subprocess:168): running <checking pip's availability>
  7206.9ms (run_subprocess:190): stdout: ModuleSpec(name='pip', loader=<_frozen_importlib_external.SourceFileLoader object at 0x7614fe5ffc50>, origin='/home/ubuntu/.local/share/pipx/shared/lib/python3.12/site-packages/pip/__init__.py', submodule_search_locations=['/home/ubuntu/.local/share/pipx/shared/lib/python3.12/site-packages/pip'])
  7206.9ms (run_subprocess:193): returncode: 0
  7207.0ms (run_subprocess:168): running <checking pip's availability>
  7225.6ms (run_subprocess:190): stdout: ModuleSpec(name='pip', loader=<_frozen_importlib_external.SourceFileLoader object at 0x739f279ffc50>, origin='/home/ubuntu/.local/share/pipx/shared/lib/python3.12/site-packages/pip/__init__.py', submodule_search_locations=['/home/ubuntu/.local/share/pipx/shared/lib/python3.12/site-packages/pip'])
  7225.7ms (run_subprocess:193): returncode: 0
  7227.0ms (warn_if_not_on_path:466): ⚠️  Note: '/home/ubuntu/.local/bin' is not on your PATH environment variable. These apps will not be globally accessible until your
    PATH is updated. Run `pipx ensurepath` to automatically add it, or manually modify your PATH in your shell's config file (i.e.
    ~/.bashrc).
  7227.4ms (cli:932): pipx finished.
